packages/flutter/lib/src/widgets/editable.dart

```diff
@@ -318,6 +348,9 @@ class RawInputLineState extends ScrollableState<RawInputLine> {
       _keyboardHandle.release();
     if (_cursorTimer != null)
       _stopCursorTimer();
+    Timer.run(() { // can't hide while disposing, since it triggers a rebuild
```

Comment:
scheduleMicrotask rather than Timer.  You want this to run before we process any more events.